### ISSUE TITLE
Support multiple methods for loading a swagger document.

### ISSUE THREAD

**Issue by ashoda** [westfieldlabs/apivore#100]
There are various requests around supporting loading swagger docs by different means. 
**Current behavior**
- Load JSON from a path relative to the application - 

**Requested:**
- Load YAML from a path relative to the application
- Load JSON from local file
- JSON string

It'd make sense for us to look at the various request to determine the ideal approach to makes loading a swagger doc a bit more flexible.

My initial thought is to add support for a Hash or a JSON string in addition to what we currently support. Which should afford people enough flexibility to decide how they'd like to load the swagger doc.

**Comment by jreece1567** [comment]
@ashoda Agreed. I think Apivore should support **both** YAML and JSON, loaded in all of the requested ways ... 

- From a path relative to the application (an HTTP/S path)
- From a local file
- From a string

cc: @gwshaw

**Comment by ben-axnick** [comment]
@ashoda I have proposed https://github.com/westfieldlabs/apivore/pull/106 as a solution.

**Comment by philsturgeon** [comment]
Kinda taken care of in #118 which adds YAML, as it also pulls in #87 which supports local files.
